frontend/packages/console-shared/src/utils/validation/validation.ts

```diff
@@ -44,7 +45,11 @@ export const validateEmptyValue = (
 // DNS-1123 subdomain
 export const validateDNS1123SubdomainValue = (
   value: string,
-  { subject } = { subject: 'Name' },
+  { subject, min, max }: { subject: string; min?: number; max?: number } = {
```

Comment:
this will not default to DNS1123_MAX_LENGTH when empty object is passed. It should always default this value. For example this validation stoped working for basic wizard

Reply:
Fixed!